requirements/ci.txt

```diff
@@ -12,25 +12,25 @@ charset-normalizer==2.0.7
     # via requests
 codecov==2.1.12
     # via -r requirements/ci.in
-coverage==6.0.2
+coverage==6.1.1
     # via codecov
 distlib==0.3.3
     # via virtualenv
-filelock==3.3.1
+filelock==3.3.2
     # via
     #   tox
     #   virtualenv
 idna==3.3
     # via requests
-packaging==21.0
+packaging==21.2
     # via tox
 platformdirs==2.4.0
     # via virtualenv
 pluggy==1.0.0
     # via tox
 py==1.10.0
     # via tox
-pyparsing==3.0.1
+pyparsing==2.4.7
```

Comment:
`packaging==21.2` constrained the `pyparsing<3.0.0` in this [PR](https://github.com/pypa/packaging/pull/471) which caused the dowgrading of this package.